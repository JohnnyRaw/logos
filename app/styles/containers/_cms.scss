.app__cms {
  &__img {
    align-items: center;
    display: flex;
    height: 10rem;
    justify-content: center;
    width: 10rem;

    a {
      display: block;
    }

    img {
      max-height: 10rem;
    }
  }

  &__files {
    a {
      color: #999;
      display: block;
    }
  }

  &__tags,
  &__categories {
    display: flex;
    flex-wrap: wrap;

    a {
      background-color: lighten($brand-primary, 45);
      border: 0.1rem solid lighten($brand-primary, 40);
      border-radius: 0.2rem;
      color: $brand-primary;
      display: block;
      font-size: $font-size-sm;
      line-height: 1;
      margin-bottom: $grid-gutter-width / 6;
      margin-right: $grid-gutter-width / 6;
      padding: $grid-gutter-width / 6;
    }
  }

  &__categories {
    margin-bottom: $grid-gutter-width / 3;

    a {
      background-color: $brand-primary;
      border-color: $brand-primary;
      color: #fff;
    }
  }

  &__toolbar {
    align-items: center;
    display: flex;
    margin-bottom: $grid-gutter-width / 3;
  }

  h4 a {
    color: $body-color;
  }

  .footable-header {
    th:last-of-type {
      @include ix {
        width: 10rem;
      }
    }
  }
  
  .app__header {
    a {
      color: #fff;
    }
  }
}
