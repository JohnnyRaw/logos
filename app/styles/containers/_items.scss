.app__items {
  position: relative;
  z-index: 10;
}
