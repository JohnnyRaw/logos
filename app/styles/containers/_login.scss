.app__login {
  .app__container {
    position: relative;
  }

  &__wrapper {
    align-items: center;
    display: flex;
    flex-direction: column;
    height: calc(100vh - #{$main-padding});
    justify-content: center;

    > div {
      font-size: $font-size-h3;
      margin-top: $grid-gutter-width;
    }

    a + a {
      margin-top: $grid-gutter-width / 2;
    }
  }
}
