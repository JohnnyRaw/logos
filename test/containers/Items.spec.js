import React from 'react';
import { shallow } from 'enzyme';

import { Items } from 'containers/Items';
import { appState } from 'reducers/app';
import { firebaseState } from 'reducers/firebase';

const mockDispatch = jest.fn();

function setup() {
  const props = {
    app: appState,
    dispatch: mockDispatch,
    firebase: firebaseState,
    location: {}
  };

  return shallow(<Items {...props} />);
}

describe('Items', () => {
  const wrapper = setup(true);

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
