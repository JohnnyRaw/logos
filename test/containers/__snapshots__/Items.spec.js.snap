exports[`Items should render properly 1`] = `"<div class=\"app__items app__route\"></div>"`;
